public/static/cql/mCODE.cql

```diff
@@ -4,9 +4,121 @@ using FHIR version '4.0.0'
 
 // CODESYSTEMS
 codesystem "SNOMEDCT": 'http://snomed.info/sct'
+codesystem "LOINC": 'http://loinc.org'
+codesystem "OBSCAT": 'http://terminology.hl7.org/CodeSystem/observation-category'
+
+// FHIR CODES
+code "Primary cancer code": '64572001' from "SNOMEDCT" display 'Disease (disorder)'
+code "TNM clinical primary tumor code": '21905-5' from "LOINC" display 'Primary tumor.clinical [Class] Cancer'
+code "TNM clinical regional nodes code": '21906-3' from "LOINC" display 'Regional lymph nodes.clinical [Class] Cancer'
+code "TNM clinical distant metastases code": '21907-1' from "LOINC" display 'Distant metastases.clinical [Class] Cancer'
+code "Tumor marker test code": 'laboratory' from "OBSCAT"
+code "Estrogen receptor code": '85337-4' from "LOINC" display 'Estrogen receptor Ag [Presence] in Breast cancer specimen by Immune stain'
+code "Progesterone receptor code": '85339-0' from "LOINC" display 'Progesterone receptor Ag [Presence] in Breast cancer specimen by Immune stain'
+code "HER2 receptor code": '85319-2' from "LOINC" display 'HER2 [Presence] in Breast cancer specimen by Immune stain'
+
+// selected cancer codes used within Synthea
+// not the entire list supported by PrimaryCancerConditionVS
+code "11p partial monosomy syndrome 4135001 code": '4135001' from SNOMEDCT display '11p partial monosomy syndrome'
```

Comment:
@dehall I think you wrote these originally for the criteria? All of the `code`s include "code" at the end but as for including the numerical code I'm not sure why

Reply:
I copied them into the mCODE_Library.cql file from an old version of the CQL on our internal repo: https://gitlab.mitre.org/mcode/pathways/-/blob/master/src/model/valuesets/PrimaryCancerVS.cql

That said I'm fine with whatever format we want to use for the code names, because we should only be referencing them internally in the definition of a concept.

Reply:
I removed the numerical code values for consistency with the rest of the file.
Ie: "11p partial monosomy syndrome 4135001 code" --> "11p partial monosomy syndrome code"